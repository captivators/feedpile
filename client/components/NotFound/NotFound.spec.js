import React from 'react';
import { shallow, render } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import NotFound from './NotFound'

test('NotFound take a snapshot', () => {
  const component = shallow(<NotFound />);
  const tree = shallowToJson(component);
  expect(tree).toMatchSnapshot();
});
