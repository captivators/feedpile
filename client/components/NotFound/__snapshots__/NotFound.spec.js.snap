// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound take a snapshot 1`] = `
<div>
  <div>
    <Link
      replace={false}
      to="/"
    >
      Home
    </Link>
  </div>
  <div>
    <Link
      replace={false}
      to="/about"
    >
      About
    </Link>
  </div>
  <hr />
  <h1>
    404 
    <small>
      Not Found :(
    </small>
  </h1>
</div>
`;
